pam/deploy_template.yaml

```diff
@@ -0,0 +1,255 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: approval-pam
+objects:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: approval-pam
+    labels:
+      app: approval
+    annotations:
+      description: All the KIE server web server's ports.
+  spec:
+    ports:
+    - name: approval-pam
+      port: 8080
+      targetPort: 8080
+    selector:
+      name: approval-pam
+- kind: Service
+  apiVersion: v1
+  spec:
+    clusterIP: "None"
+    ports:
+    - name: "ping"
+      port: 8888
+      targetPort: 8888
+    selector:
+      name: approval-pam
+  metadata:
+    name: approval-pam-ping
+    labels:
+      app: approval
+    annotations:
+      service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
```

Comment:
After looking around a bit, it seems like support for this was removed from kubernetes at least a few releases ago. We I think we can probably remove this whole annotations section.